<?xml version="1.0"?>
<!-- CALIBRATION ID: 1715240213 -->
<launch>

  <!-- Network parameters -->

  <!-- pc: AX-Lab2005 -->
  <!-- sensor: realsense2 -->
  <node pkg="tf" type="static_transform_publisher" name="realsense2_broadcaster"
        args="-0.762108 1.09533 1.42939 0.536197 0.756396 -0.177949 0.329685 /world /realsense2 100" />


  <!-- sensor: realsense3 -->
  <node pkg="tf" type="static_transform_publisher" name="realsense3_broadcaster"
        args="0.137945 1.09102 1.14799 0.607716 0.685889 -0.363188 0.16832 /world /realsense3 100" />


</launch>
